Cargo.toml

```diff
@@ -9,58 +9,51 @@ edition = "2021"
 include = ["src/**/*", "LICENSE-*", "*.md"]
 
 [dependencies]
-clap = { version = "~4.5", features = ["derive", "std", "help"], default-features = false }
-git2 = { version = "~0.19", features = ["ssh", "https"], default-features = false }
+anstyle = "1.0.7"
+anyhow = "~1.0"
 auth-git2 = "~0.5"
+clap = { version = "~4.5", features = ["derive", "wrap_help"] }
```

Comment:
This is the only real change in this file (adding wrap help and turning on default features, which activates the color and the usage features).

There's no specific reason in the git / PR history why default features was turned off here for clap. See https://github.com/cargo-generate/cargo-generate/pull/655

Reply:
fair point.